pom.xml

```diff
@@ -181,10 +189,17 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.0</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <release>${java.version}</release>
                 </configuration>
+                <dependencies>

```

Comment:
Do we need a dependency to https://asm.ow2.io/? Why?

Reply:
Let me check this in a clean environment(not my slt machine) because when I executed it, Maven need to know the exact bytecode. Thanks!

Reply:
I've checked springboot app in a clean environment whitout the OSM bytecode dependency, and it works perfectly. So, I've removed the OSM dependency.

Thx!